Quickstore/pages/wishlist.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>QuickStore | Wishlist</title>
+
+    <!---------component libaray style-sheet ------>
+    <link rel="stylesheet" href="https://boostweb-ui.netlify.app/component/boostweb.css">
+    
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> 
```

Comment:
This can be moved into your main component.css so you don't have to include it in every page

Reply:
ok I will consider it